pkg/networkgraph/tree/radix.go

```diff
@@ -456,6 +460,79 @@ func (t *nRadixTree) validateCardinality() bool {
 	return getCardinalityByValues(t.root) == t.Cardinality()
 }
 
+func cloneIPNet(ipNet *net.IPNet) *net.IPNet {
+	ip := make(net.IP, len(ipNet.IP))
+	copy(ip, ipNet.IP)
+
+	mask := make(net.IPMask, len(ipNet.Mask))
+	copy(mask, ipNet.Mask)
+
+	return &net.IPNet{
+		IP:   ip,
+		Mask: mask,
+	}
+}
+
+func compareValueIpNet(value *storage.NetworkEntityInfo, ipNet *net.IPNet) bool {
```

Comment:
How about naming it, so that the name uses the word `equals`?
Compare slightly suggests that returned value is -1,0,1.